.templates/template_location.json

```diff
@@ -4,83 +4,103 @@
     "directoryPath": "python/python-hello-world",
     "templatePath": "python-hello-world",
     "templateName": "Python (Flask): Hello World",
-    "templateLanguages": ["python"]
+    "templateLanguages": ["python"],
+    "run-platform": ["kubernetes"]
```

Comment:
Just a suggestion (since we're consuming this for the IntelliJ plugin), you can rename this field from `run-platform` to `runPlatforms` (plural for platforms since this is a list, and removing the dash, which may cause issues when serializing/deserializing)

Reply:
+1. and regardless of consuming platform, the format should be consistent across these fields.

also +1 to making it plural since its an array type

Reply:
np, changing it to runPlatforms